tests/lint/files_exist.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+
+import os
+import yaml
+import nf_core.lint
```

Comment:
Would be good to add a test that passes too 👍🏻

Reply:
👍